smt_switch/src/solvers/BoolectorSolver.py

```diff
@@ -92,25 +93,26 @@ def set_nonstandard_option(self, optionstr, value):
     def declare_const(self, name, sort):
```

Comment:
Unfortunately I don't have a declare-fun equivalent yet, that needs to be added still.